src/vs/workbench/contrib/terminal/browser/terminalFindWidget.ts

```diff
@@ -127,3 +146,11 @@ export class TerminalFindWidget extends SimpleFindWidget {
 		}
 	}
 }
+
+registerThemingParticipant((theme, collector) => {
+	const error = theme.getColor(errorForeground);
```

Comment:
Is this the same key used for the editor's find?

Reply:
yes